### PR TITLE
Localization

### PR DESCRIPTION
**Pull request by Kabak-Siarhei**
Fixed Switch
Added localization

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -1,31 +1,44 @@
...
 }
 
 class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return BlocProvider(
-      create: (context) => ThemeBloc(),
-      child: BlocBuilder<ThemeBloc, ThemeChangeState>(
-        builder: (context, state) {
+    return MultiProvider(
```
Comment (shcherbuk96) [inline lib/main.dart:17]: Why did you use MultiProvider?
Could you please change it to BlocProvider?

File: lib/main.dart [hunk 2]
```diff
@@ -1,31 +1,44 @@
...
+        BlocProvider<ThemeBloc>(
+          create: (context) => ThemeBloc(),
+        )
+      ],
+      child:
+          BlocBuilder<ThemeBloc, ThemeChangeState>(builder: (context, state) {
         bool isDarkTheme = false;
         if (state is ThemeChangeState) {
           isDarkTheme = state.isDarkTheme;
         }
```
Comment (shcherbuk96) [inline lib/main.dart:28]: Add empty line

File: lib/main.dart [hunk 3]
```diff
@@ -1,31 +1,44 @@
...
+    return MultiProvider(
+      providers: [
+        BlocProvider<ThemeBloc>(
+          create: (context) => ThemeBloc(),
+        )
+      ],
+      child:
+          BlocBuilder<ThemeBloc, ThemeChangeState>(builder: (context, state) {
         bool isDarkTheme = false;
         if (state is ThemeChangeState) {
```
Comment (shcherbuk96) [inline lib/main.dart:26]: Add empty line

File: pubspec.yaml [hunk 4]
```diff
@@ -23,12 +23,15 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-
+  flutter_localizations:
+    sdk: flutter
+  flutter_bloc: ^7.1.0
+  provider:
```
Comment (shcherbuk96) [inline pubspec.yaml:29]: For what did you use provider? And why is it empty?

File: lib/main.dart (hunk 2 above)
Comment (Kabak-Siarhei) [reply lib/main.dart:24]: Done

File: lib/main.dart (hunk 3 above)
Comment (Kabak-Siarhei) [reply lib/main.dart:22]: Done

File: lib/main.dart (hunk 1 above)
Comment (Kabak-Siarhei) [reply lib/main.dart]: > Why did you use MultiProvider?

I think in the future I need necessary to transfer the variables of this screen to other screen. 

> Could you please change it to BlocProvider?
Done

File: pubspec.yaml (hunk 4 above)
Comment (Kabak-Siarhei) [reply pubspec.yaml]: > For what did you use provider?

For widget MultiProvider

> And why it is empty?

I forgot to write a version.

**Pull request merged**